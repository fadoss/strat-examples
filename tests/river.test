modelChecking/river

srew initial using safe .
srew initial using eagerEating .
