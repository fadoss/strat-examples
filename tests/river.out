==========================================
srewrite in RIVER-CHECK : initial using safe .

Solution 1
rewrites: 33
result River: left | shepherd wolf goat cabbage right

No more solutions.
rewrites: 33
==========================================
srewrite in RIVER-CHECK : initial using eagerEating .

Solution 1
rewrites: 71
result River: left | shepherd wolf goat cabbage right

No more solutions.
rewrites: 71
Bye.
